components/lists/src/css/stacked-list.css

```diff
@@ -0,0 +1,21 @@
+@import url(../../../../css/component.css);
+.stacked-list .basic-card {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+  padding: 1rem;
+  background-color: var(--font-color);
+  color: var(--background-color);
+  border-radius: 1rem;
+}
+
+.stacked-list li {
+  margin: 2rem;
+}
```

Comment:
Give spacing between line number 14 and 15.

Reply:
Duplicates are there to maintain folder structure, they will be deleted once completed with the CL